web/src/components/player/LivePlayer.tsx

```diff
@@ -126,7 +126,7 @@ export default function LivePlayer({
     <div
       className={`relative flex justify-center w-full outline ${
         activeTracking
-          ? "outline-destructive outline-1 rounded-2xl shadow-[0_0_6px_1px] shadow-destructive"
```

Comment:
is this removing the red from the live player outline?

Reply:
good catch, forgot to add back in those custom tailwind classes